# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/nsBaseWidget.cpp
# Commit: 425313809206
# Full Hash: 425313809206add668132f5653c0bba3a91e79d5
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-24 09:47:24
# Description:
#   Bug 1726562 - Remove option to disable software webrender for popups. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123159
# ==============================================================================

diff -r 60b0cb3f37fe -r 425313809206 widget/nsBaseWidget.cpp
--- a/widget/nsBaseWidget.cpp	Tue Aug 24 03:29:21 2021 +0000
+++ b/widget/nsBaseWidget.cpp	Tue Aug 24 04:06:34 2021 +0000
@@ -1336,12 +1336,8 @@
         StaticPrefs::gfx_webrender_unaccelerated_widget_force()) {
       enableWR = gfx::gfxVars::UseWebRender();
       enableSWWR = gfx::gfxVars::UseSoftwareWebRender();
-    } else if (gfxPlatform::DoesFissionForceWebRender() ||
-               StaticPrefs::
-                   gfx_webrender_software_unaccelerated_widget_allow()) {
+    } else {
       enableWR = enableSWWR = gfx::gfxVars::UseWebRender();
-    } else {
-      enableWR = enableSWWR = false;
     }
     MOZ_RELEASE_ASSERT(enableWR);
     bool enableAPZ = UseAPZ();
